Shared/sdk/net/bitstream.h

```diff
@@ -608,6 +608,10 @@ enum class eBitStreamVersion : unsigned short
     // 2025-01-10
     WorldSpecialProperty_FlyingComponents,
 
+    //2025-01-16
+    //Add isVehicleNitroActivated and getVehicleNitroLevel Server side
```

Comment:
Maintain consistency with other entries to avoid creating a mess

Reply:
what do you mean

Reply:
```
// Comment
// Date
```

![image](https://github.com/user-attachments/assets/180cf815-53b7-46a5-877b-c7d44985d87d)